src/test/java/gov/cms/mat/cql_elm_translation/utils/ResourceUtilsTest.java

```diff
@@ -13,14 +13,16 @@
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
 
-import gov.cms.mat.cql_elm_translation.exceptions.InternalServerException;
+import gov.cms.madie.cql_elm_translator.exceptions.InternalServerException;
+import gov.cms.madie.cql_elm_translator.utils.ResourceUtils;
+import gov.cms.madie.cql_elm_translator.utils.StreamUtils;
 
 @ExtendWith(MockitoExtension.class)
 public class ResourceUtilsTest {
 
   @Test
   public void testReadData() {
-    String result = ResourceUtils.getData("/cv_populations.cql");
+    String result = new ResourceUtils().getData("/cv_populations.cql");
```

Comment:
is this not a static method? Can we not use `ResourceUtils.getData`?

Reply:
@adongare Yeah I just had a couple of minor changes, please have another review, thanks so much!!